dashboard/db/migrate/20170522221144_add_course_id_to_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseIdToSections < ActiveRecord::Migration[5.0]
+  def change
+    add_column :sections, :course_id, :integer
```

Comment:
Any reason not to add `after: code` so that the `course_id` and `script_id` are adjacent?

Any reason this should / shouldn't be `t.references :script` (as part of a `change_table`) so as to have a foreign key constraint?